runtime/drivers/catalog.go

```diff
@@ -36,6 +36,7 @@ type CatalogEntry struct {
 	Object      proto.Message
 	Path        string
 	Embedded    bool
+	SizeInBytes int64
```

Comment:
I didn't put it in `runtimev1.Source` since the comment on it mentions that its just an internal representation of source definition. In that case we would not want to keep some internal data in that. 
I think it makes sense to keep a one to one mapping between source yaml and `runtimev1.Source`. Please let me know your thoughts.

Reply:
You are right. The reconcile refactor will introduce a new `State` field, so we can have `SourceState`, `ModelState`, etc. That's where it should really belong. Until we do that, I'm fine with either abusing this table or the `runtimev1.Source` type.